<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Biodiversity data - from Field to Yield</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Exercises
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="01_download_GBIF_data_exercise.html">01 - Downloading data from GBIF</a>
    </li>
    <li>
      <a href="02_data_cleaning_exercise">02 - Data cleaning</a>
    </li>
    <li>
      <a href="03_sampling_biases_exercise.html">03 - Sampling bias detection</a>
    </li>
    <li>
      <a href="04_species_richness_maps_exercise.html">04 - Richness maps</a>
    </li>
    <li>
      <a href="05_ range_size_and_conservation_assessment_exercise.html">05 - Range size and conservation assessment</a>
    </li>
    <li>
      <a href="06_taxon-specific bioregionalization_exercise.html">06 - Bioregionalization</a>
    </li>
    <li>
      <a href="07_biome_classification_exercise.html">07 - Biome classification</a>
    </li>
    <li>
      <a href="08_maps and phylogenetic visualization_exercise.html">08 - Visualization</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="01_download_GBIF_data_tutorial.html">01 - Downloading data from GBIF</a>
    </li>
    <li>
      <a href="02_data_cleaning_tutorial">02 - Data cleaning</a>
    </li>
    <li>
      <a href="03_sampling_biases_tutorial.html">03 - Sampling bias detection</a>
    </li>
    <li>
      <a href="04_species_richness_maps_tutorial.html">04 - Richness maps</a>
    </li>
    <li>
      <a href="05_ range_size_and_conservation_assessment_tutorial.html">05 - Range size and conservation assessment</a>
    </li>
    <li>
      <a href="07_biome_classification_tutorial.html">07 - Biome classification</a>
    </li>
    <li>
      <a href="08_maps and phylogenetic visualization_tutorial.html">08 - Visualization</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/azizka/Biodiversity_Data_from_Field_to_Yield/">GitHub</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="master-class-biodiversity_data_from_field_to_yield" class="section level1">
<h1>Master class: Biodiversity_Data_from_Field_to_Yield</h1>
<p>Date &amp; time: Tuesday, October 2 â€“ Saturday, October 6, 2018 (5 full days)</p>
<p>Location: Programa de PÃ³s-GraduaÃ§Ã£o em SistemÃ¡tica e EvoluÃ§Ã£o, Universidade Federal do Rio Grande do Norte, Natal, Brazil</p>
<p>Teachers: Alexandre Antonelli &amp; Alexander Zizka, University of Gothenburg, Gothenburg Global Biodiversity Centre, and German Centre for Integrative Biodiversity Research</p>
<p><a href="http://antonelli-lab.net" class="uri">http://antonelli-lab.net</a> <a href="http://ggbc.gu.se" class="uri">http://ggbc.gu.se</a> <a href="https://www.idiv.de/" class="uri">https://www.idiv.de/</a></p>
</div>
<div id="before-the-course" class="section level1">
<h1>Before the course</h1>
<p>Please:</p>
<ul>
<li><p>Read the course <a href="https://github.com/azizka/Biodiversity_Data_from_Field_to_Yield#literature">literature</a></p></li>
<li><p>Download the <a href="https://knowme.earth/">knowme.earth app</a> to your phone</p></li>
<li><p>Install <a href="https://cran.r-project.org/bin/">R</a> and <a href="https://www.rstudio.com/products/rstudio/download/#download">Rstudio</a> on your computer</p></li>
<li><p>Open R studio and install the necessary dependencies by copying the code below into the R console and running it.</p></li>
</ul>
<p>Please do not hesitate to <a href="mailto:zizka.alexander@gmail.com">contact us</a> if you have any questions.</p>
</div>
<div id="installation" class="section level1">
<h1>Installation</h1>
<p>Please run the following code (just copy it into the R console) before the course to make sure all dependencies are installed and we can start smoothly.</p>
<pre class="r"><code>install.packages(c(&quot;countrycode&quot;, &quot;ConR&quot;, &quot;devtools&quot;, &quot;ggmap&quot;, &quot;mapproj&quot;, &quot;maps&quot;,
                   &quot;rgbif&quot;, &quot;raster&quot;, &quot;rnaturalearth&quot;, &quot;sp&quot;, &quot;tidyverse&quot;,
                   &quot;viridis&quot;))

library(devtools)

install_github(repo = &quot;azizka/speciesgeocodeR&quot;)
install_github(repo = &quot;azizka/sampbias&quot;)</code></pre>
</div>
<div id="objectives" class="section level1">
<h1>Objectives</h1>
<ol style="list-style-type: decimal">
<li><p>Exemplify the various uses of biodiversity data for exploratory and question-driven research in ecology, evolution and biogeography</p></li>
<li><p>Familiarize participants with new bioinformatic tools for handling and processing â€˜big dataâ€™, including dealing with data errors and biases</p></li>
<li><p>Provide the participants with a workflow to use large scale species occurrence data for biodiversity analyses (including point localities and range maps)</p></li>
<li><p>Provide an overview of how to use R for analysing large datasets of species occurrences (including data mining, visualization, exploration, cleaning and applications)</p></li>
</ol>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<p>The public availability of large-scale species distribution data has increased drastically over the last ten years. In particular, the digitalization of collections from museums and herbaria, the input from human and machine observations, and the aggregation of information in public databases such as the Global Biodiversity Information Facility (GBIF) have contributed significantly to this development. This is leading to a â€˜big dataâ€™ revolution in biogeography, which holds an enormous but still poorly explored potential for understanding large scale patterns and drivers of biodiversity.</p>
</div>
<div id="project-assignment" class="section level1">
<h1>Project Assignment</h1>
<p>During the course you will pick a taxonomic group of your interest and then independently collect occurrence data and answer biogeographic questions on this group with guidance from the teachers.</p>
</div>
<div id="software-exercises-and-project-questions" class="section level1">
<h1>Software exercises and Project questions</h1>
<p>After collecting occurrence records in the field we will spend day four of the course getting to know different software to process the data and answer the research questions of your projects. There are seven exercises, each one representing a step to gather and process geographic occurrence information to tackle questions of your research project.</p>
<p>Most of the exercises are in R, although exercise 1) and 6) can also be done using a web browser. Each exercise comes as a set of questions with some suggestions on how to solve them. If necessary, example answers are available to guide the students. Exercises 1) and 2) are compulsory, beyond that you can chose which exercises to follow based on the questions of your project.</p>
</div>
<div id="examination" class="section level1">
<h1>Examination</h1>
<p>Grades are pass/fail. Successful participants should participate in all course days and present a project on the last day. A certification will be issued for all participants.</p>
</div>
<div id="literature" class="section level1">
<h1>Literature</h1>
<ol style="list-style-type: decimal">
<li><p>Meyer <em>et al.</em> (2016) Multidimensional biases, gaps and uncertainties in global plant occurrence information. <em>Ecology Letters</em> 19:992-1006.</p></li>
<li><p>Schmidt <em>et al.</em> (2017) Diversity, distribution and preliminary conservation status of the flora of Burkina Faso. Magnolia Press.</p></li>
<li><p>Antonelli <em>et al.</em> (2018) Amazonia is the primary source of Neotropical biodiversity. PNAS 6pp</p></li>
<li><p>One of the following suggestions (depending on your own interests):</p></li>
</ol>
<ul>
<li><p>Edler <em>et al.</em> (2017) Infomap Bioregions: Interactive mapping of biogeographical regions from species distributions. <em>Systematic Biology</em> 66(2):197â€“204</p></li>
<li><p>Antonelli (2017) Comparative biogeography, big data, and common myths. In: <em>Tropical Plant Collections: Legacies from the Past? Essential Tools for the future. Scientia Danica. Series B, Biologica. vol. 6</em></p></li>
<li><p>Silvestro <em>et al.</em>(2016) Fossil biogeography: a new model to infer dispersal, extinction and sampling from palaeontological data. <em>Philosophical Transactions of the Royal Society B</em> 371:20150225</p></li>
<li><p>Price <em>et al.</em> (2019) Big data little help in megafauna mysteries. Nature 558(7):23-25</p></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
